charts/super-agent/templates/uninstall-job.yaml

```diff
@@ -23,10 +28,13 @@ spec:
         - name: delete-crs
           image: bitnami/kubectl
           command:
-            - kubectl
-            - delete
-            - -f
-            - /manifests/
+            - bash
+          args:
```

Comment:
shouldn't be part of the `super-agent-deployment` ?

Reply:
After properly testing it, it is not needed here. As long as we assure that deleting these manifest will wait for the underlying chart to be uninstalled there should not be race conditions here.